### ISSUE TITLE
[1.9.4] Description on Armor

### ISSUE THREAD

**Issue by MrAmericanMike** [sokratis12GR/ArmorPlus#25]
More than an issue, it's a suggestion. When someone change on the configs, for example.
On Coal Armor

```

# Enable/Disable The Coal Boots NightVision [default: true]
    B:enableCoalBNightVision=false

```

The description for that piece, still says "Gives you night vision"

So here my idea, to make it only show that if the config is true and also, in the case we have 

```

# Enable/Disable The Full Coal Armor Effect [default: false]
    B:enableFullCoalArmorEffect=true

```

Make it so it says "Gives you night vision when using full set"

PS: What I'm not sure how would it be handled, when you have 1 piece giving the effect, and also the full set set to true, I guess that wont make sense as you would have full set, but you already getting the effect for the 1 piece left on true on the configs ;)

**Comment by sokratis12GR** [comment]
Nice idea dude :D 
This is going to be a feature in the next update

**Comment by MrAmericanMike** [comment]
Thanks for the implementation. I tested it on the newer versions, and it's "almost" perfect. The only catch is that when you have for example, all the Emerald effect's turn off (false) and only set to true B:enableFullEmeraldArmorEffect=true the description for the items still say they give you the effect, without saying "When full set is equipped" Other than that, working great.

PS: The items on the Creative tab got scattered? Or maybe it's just because I updated and launched again on same world?
